frontend/src/metabase/query_builder/components/view/ViewHeader/ViewTitleHeader.module.css

```diff
@@ -15,42 +15,26 @@
     transition: background 300ms linear;
 
     &:hover {
-      color: var(--mb-color-filter);
       background-color: color-mix(in srgb, var(--mb-color-filter) 10%, white);
-
-      .FilterCountChip {
-        background-color: var(--mb-color-filter);
-      }
     }
 
+    &:hover,
```

Comment:
I don't love that the hover state and active state are the same because it makes it so that there's no visual feedback when the button is clicked.

See [this Loom I recorded to explain what I mean](https://www.loom.com/share/564dfa2df960437bb5cac094da9e7330)